### PR TITLE
Update width vimeo's video and css design for journal's preview

### PR DESCRIPTION
**Pull request by EEUAAAIA**
Adding percentage for the width in vimeo iframe tag and I replaced the p tags with div tags.

### CODE REVIEW

File: app/views/default/pages/journal_items.html.erb [hunk 1]
```diff
@@ -11,15 +11,15 @@
         <div class="journal__hero-image">
           <%= image_tag original_url(current_page.content(:journal_image).spina_image) %>
         </div>
-        <p class="perex">
```
Comment (bolom) [inline app/views/default/pages/journal_items.html.erb:14]: don't change the HTML struture, HTML versions works with p not div

Comment (EEUAAAIA) [reply app/views/default/pages/journal_items.html.erb:14]: I put the p tag back with the perex class. To display the content of the tag I used strip_tags to display only the text without the HTML formatting: <%= strip_tags(current_page.content(:journal_preview_description)) %>

Comment (bolom) [reply app/views/default/pages/journal_items.html.erb:14]: I don't really like that, it's look overcomplicated. why you don't use 

<%= content.html :body %>

Please use the code exemple as reference

https://spinacms.com/code-examples

Comment (EEUAAAIA) [reply app/views/default/pages/journal_items.html.erb:14]: I tried it : <%= content.html :journal_preview_description %> but that doesn't work.

**Review by bolom — APPROVED** [review]

**Pull request merged**